src/test/java/org/jabref/logic/bibtexkeypattern/BibtexKeyGeneratorTest.java

```diff
@@ -879,15 +925,15 @@ void keywordNKeywordsSeparatedBySpace() {
         BibEntry entry = new BibEntry();
         entry.setField(StandardField.KEYWORDS, "w1, w2a w2b, w3");
 
-        String result = BibtexKeyGenerator.generateKey(entry, "keyword1");
+        String result = generateKey(entry, "keyword1");
```

Comment:
In other test cases you inline the argument for assertEquals.

Reply:
Yes. I don't have much experience with Java programming, therefore I mostly leave the original code unchanged. I try "pushing my luck" with readability "improvements" in `BrackettedPattern.generateInstitutionKey`(lines 1312-1429) as a way to balance improving my own Java programming knowledge without adding too much extra work for reviewers.

AuthorListTest lines 39-41 is another example of where I have left previous code untouched while I'd like to change it to use `assertSame`.

Right now, I view my PRs to JabRef as a balancing act. If I am being too cautious or bold, feel free to tell me. I'd be much happier if I leave JabRef's codebase better of without wasting too much time of reviewers :P